Sources/LaunchItem.swift

```diff
@@ -72,101 +93,77 @@ public enum LaunchItem: CustomStringConvertible, Equatable {
     public var description: String {
         // Creating a custom string for LaunchItem to prevent a crash when printing any enum values with associated items on iOS 8. Swift accesses the class of every possible associated object when printing an enum instance with an asssociated value, no matter which case the instance represents. This causes a crash on iOS 8 since swift attempts to access UIApplicationShortcutItem, which doesn't exist on iOS 8. Filed rdar://26699861 – hoping for a fix in Swift 3.
```

Comment:
didn't check. Can't hurt to leave it in though or follow up later. Good q though!